### ISSUE TITLE
[Request] Pretrained Download Links

### ISSUE THREAD

**Issue by bomcon123456** [Zhendong-Wang/Patch-Diffusion#1]
Thank you for your awesome work! I'm very interested in the paper. I saw that you planned to publish the pre-trained model, will you upload it soon? I'm very excited to test the models tbh.
Thanks again!

**Comment by Zhendong-Wang** [comment]
Yes, the works has been accepted by NeurIPS 2023, and we will share the pre-trained models pretty soon.

**Comment by Zhendong-Wang** [comment]
Sorry for the delay due. Checkpoints will be updated to this link https://huggingface.co/zhendongw/patch-diffusion/tree/main. Now ckpts for ImageNet, LSUN-Bedroom and LSUN-Church are shared.
